src/Pim/Bundle/CatalogBundle/tests/integration/Elasticsearch/PimCatalogTextIndexConfigurationIntegration.php

```diff
@@ -0,0 +1,298 @@
+<?php
+
+namespace Pim\Bundle\CatalogBundle\tests\integration\Elasticsearch;
+
+/**
+ * This integration tests checks that given an index configuration and some products indexed
+ * the text research are consistent.
+ *
+ * @author    Samir Boulil <samir.boulil@akeneo.com>
+ * @copyright 2017 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class PimCatalogTextIndexConfigurationIntegration extends AbstractIndexConfigurationIntegration
```

Comment:
I think this is the string phpunit is looking for to run the tests (also this is the convention used everywhere in the PIM regarding the integration tests). It will most certainly stay like that.